lib/PhasarLLVM/DataFlow/IfdsIde/Problems/IDETypeStateAnalysis.cpp

```diff
@@ -685,10 +685,12 @@ bool IDETypeStateAnalysis::hasMatchingType(IDETypeStateAnalysis::d_t V) {
                             TSD->getTypeNameOfInterest())) {
       return true;
     }
+    // fallthrough
   }
   if (const auto *Alloca = llvm::dyn_cast<llvm::AllocaInst>(V)) {
     if (Alloca->getAllocatedType()->isPointerTy()) {
-      if (hasMatchingTypeName(
+      if (Alloca->getAllocatedType()->isOpaquePointerTy() ||
+          hasMatchingTypeName(
               Alloca->getAllocatedType()->getPointerElementType(),
```

Comment:
didn't getPointerElementType() disappear?

Reply:
It's just deprecated for now, but you are right, we should probably replace it with `getNonOpaquePointerElementType()`